### PR TITLE
Allow Scan Args

### PR DESCRIPTION
**Pull request by ebram-va**
In reference to the maintainer's conversation on #9 , I think it makes sense to at least as an MVP, allow users of this action to supply args.  

We would like to use this action but it's not helpful right now without this as we need certain flags.

I think it's reasonable to re-evaluate later how you want to handle this, but it'd be useful if this was at least working in an initial version.  Open to discussion though!

### CODE REVIEW

**Comment by ebram-va** [comment]
@wbeuil

**Comment by wbeuil** [comment]
Hey @ebram-va, thanks for opening this PR. It is indeed something that we want to tackle really soon in a more Github Action idiomatic way (with a map of driftctl flags).

Before making any decision in this PR whether it'll be merged or not, can you first write a new test in the github action workflow of this repo. Just re-write the first test only with flags instead of `DCTL_` env var.

**Comment by ebram-va** [comment]
@wbeuil understood, updated, let me know how that test goes, I didn't have the ability to run it against my area since I didn't have any personal AWS creds

**Comment by ebram-va** [comment]
@wbeuil just bumping this, we're closing up the implementation we needed this for on our end, would be great to move away from our forked repo as well if possible!

File: .github/workflows/driftctl-action.yml [hunk 1]
```diff
@@ -24,6 +24,24 @@ jobs:
...
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Run driftctl
+        uses: ./
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: us-east-1
+          args: |
```
Comment (wbeuil) [inline .github/workflows/driftctl-action.yml:40]: this should be within the `with` key in the same level of `env` key.

File: .github/workflows/driftctl-action.yml [hunk 2]
```diff
@@ -24,6 +24,24 @@ jobs:
...
+            --no-version-check
+            --from tfstate://tests/terraform.tfstate
+            --driftignore tests/.driftignore
+            filter Type=='unknown_type'
```
Comment (wbeuil) [inline .github/workflows/driftctl-action.yml]: This should be `--filter`

**Review by wbeuil — CHANGES_REQUESTED** [review]
Had finally time to test your PR in a private repo because we can't use our Env variables in a forked PR. Small changes and we're good to go

File: .github/workflows/driftctl-action.yml (hunk 2 above)
Comment (ebram-va) [reply .github/workflows/driftctl-action.yml]: woops missed that, thanks!

File: .github/workflows/driftctl-action.yml (hunk 1 above)
Comment (ebram-va) [reply .github/workflows/driftctl-action.yml:40]: ugh yes, got it!

**Comment by ebram-va** [comment]
Hey @wbeuil no problem, thanks for the PR review and catches!  As you could see, I also couldn't run that test locally

**Review by wbeuil — APPROVED** [review]
Will do release thanks @ebram-va

**Pull request merged**